src/presenters/business/business-address-check-presenter.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Formats data ready for presenting in the `/business-address-check` page
+ * @module businessAddressCheckPresenter
+ */
+
+const businessAddressCheckPresenter = (data) => {
+  return {
+    backLink: { href: '/business-address-enter' },
+    cancelLink: '/business-details',
+    changeLink: '/business-address-enter',
+    pageTitle: 'Check your business address is correct before submitting',
+    metaDescription: 'Check the address for your business is correct.',
+    address: formatAddress(data.businessAddress),
+    businessName: data.businessName ?? null,
+    singleBusinessIdentifier: data.singleBusinessIdentifier ?? null,
```

Comment:
can we change this to SBI as per group slack conversation

Reply:
Thank you, all updated now :)